### PR TITLE
fixes #533 : changes to the chat message in MainActivity

### LINKED ISSUE #533: Small issue in Chat Activity
When we enter text in multiple lines in the edit text field in the chat activity we get the message as one liner in our messages like this:

16777110_10210493290283003_48589909_o

I want something like this:

16763838_10210493320043747_1005002941_o

The response of susi is not getting affected as i am using the trimmed message for that but storing the actual string in the database as entered by the user.

@jig08 should i send a pr regarding the same?

### PR DESCRIPTION
**Pull request by arp95**
made changes as shown in the issue.

### CODE REVIEW

**Comment by arp95** [comment]
@Abhi2424shek @the-dagger @jig08 please review.

**Comment by the-dagger** [comment]
@arp95 Please don't edit the default Pull request template, it's there for a reason :slightly_smiling_face:

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -1544,13 +1546,13 @@ public void setReminder(String des , String time){
...
-                    .putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, System.currentTimeMillis())
-                    .putExtra(CalendarContract.EXTRA_EVENT_END_TIME, endTime.getTimeInMillis())
-                    .putExtra(CalendarContract.Events.TITLE, des)
-                    .putExtra(CalendarContract.Events.AVAILABILITY, CalendarContract.Events.AVAILABILITY_BUSY);
-            startActivity(intent);
+        Intent intent = new Intent(Intent.ACTION_INSERT)
+                .setData(CalendarContract.Events.CONTENT_URI)
+                .putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, System.currentTimeMillis())
+                .putExtra(CalendarContract.EXTRA_EVENT_END_TIME, endTime.getTimeInMillis())
+                .putExtra(CalendarContract.Events.TITLE, des)
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Also, don't disturb the original formatting.
It causes lots of extra changes which are not needed.

**Comment by arp95** [comment]
@the-dagger please review now. made the changes.

**Comment by arp95** [comment]
@the-dagger @jig08 please review.

**Pull request merged**